autoextract_spiders/spiders/crawler_spider.py

```diff
@@ -244,7 +244,7 @@ def _rule_process_links(self, links):
         valid_links = []
         for lnk in links:
             host = urlsplit(lnk.url).netloc.lower()
-            if host in self.allowed_domains:
+            if not hasattr(self, 'allowed_domains') or host in self.allowed_domains:
```

Comment:
Seems like a minor bug in the code: if `same_origin` and `seed_urls` are [not defined](https://github.com/scrapinghub/autoextract-spiders/blob/master/autoextract_spiders/spiders/crawler_spider.py#L98-L101), the attribute might not exist.

Reply:
Good catch!